server/src/main/java/com/linecorp/centraldogma/server/internal/storage/repository/git/GitRepository.java

```diff
@@ -315,6 +318,9 @@
                 commitIdDatabase.rebuild(jGitRepository);
                 assert headRevision.equals(commitIdDatabase.headRevision());
             }
+            final Commit initialCommit = blockingHistory(Revision.INIT, Revision.INIT, ALL_PATH, 1).get(0);
```

Comment:
Could we also update `CachingRepository` to delegate `author` and `creationTimeMillis`?

Reply:
Thanks, fixed. 😉